bin/render-activity-stream-html.js

```diff
@@ -156,6 +169,7 @@ function writeFiles(name, destPath, filesMap, {html, state}, options) {
 
 const STATIC_FILES = new Map([
   ["activity-stream-debug.html", ({options}) => templateHTML(options)],
+  ["activity-stream-load-scripts.js", ({options}) => templateLoadScriptJs(options)],
```

Comment:
I think I need to add another render for the debug version of this technically, not sure if we care about that or just always use the `noscripts` version for debug?

Reply:
Hmm.. the test failures are complaining that the react.js and react-dom.js versions are unreferenced, so that means activity-stream-load-scripts.js is rendering only with the debug=true ? We definitely want the main file to be the non-dev versions. Still thinking about the overall debug situation… 🤔

Reply:
Ah yup static assumes debug… … … was that intended? 🤔 
https://github.com/mozilla/activity-stream/blob/a3f538fd0cd8b8c121b8f0b21ce89775dbecd1e9/bin/render-activity-stream-html.js#L242-L243

Reply:
yeah, I think I need to just switch it to render those separately

Reply:
I think we can work with a single load-scripts.js file that dynamically determines its debug context.

rendered debug html:
```html
<script src="load-scripts.js#debug"/>
```

rendered not debug html:
```html
<script src="load-scripts.js#not-debug"/>
```

load-scripts.js:
```js
const isDebug = document.currentScript.src…;
const scripts = [
   // NB: use the full resource to make referenced test happy
  isDebug ? "resource://…react-dev.js" : "resource://…react.js"
…
```

Alternatively just package both a load-scripts-debug.js and load-scripts.js 😉